lib/shoryuken/client.rb

```diff
@@ -9,15 +9,15 @@ def queues(name)
       end
 
       def sns
-        @sns ||= Aws::SNS::Client.new(aws_client_options(:sns_endpoint))
+        @sns ||= Aws::SNS::Client.new(aws_client_options(:sns))
       end
 
       def sns_arn
         @sns_arn ||= SnsArn
       end
 
       def sqs
-        @sqs ||= Aws::SQS::Client.new(aws_client_options(:sqs_endpoint))
+        @sqs ||= Aws::SQS::Client.new(aws_client_options(:sqs))
```

Comment:
@atyndall something that's confusing me is `debug.log` and `info.log`? What's the `logfile` you are passing to Shoryuken? 

By default the AWS logger is set to `nil`, that confuses me, how is Aws logging into `debug.log` and `info.log`.